app/Http/Controllers/Admin/CallchampionsController.php

```diff
@@ -82,11 +83,44 @@ public function list_mothers($cc_id = -1){
 	public function update_call($due_id_encrypted){
 		$helper_obj = new Helpers;
 		$dueid = $helper_obj->decode($due_id_encrypted);
+		$duelist = DueList::where('due_id',$dueid)->get();
+		$b_id = $duelist[0]['fk_b_id'];
 		$action_items = Input::get('action_item');
 		$call_stats = Input::get('callstats');
 		$general_note = Input::get('general_note');
 		$cc = new CallChampion;
-		$update_status = $cc->update_cc_report($dueid, $call_stats, $general_note, $action_items);
+
+		$data['callchampion'] = User::where('user_id',$this->user_id)->get();
+		$b_obj = new Beneficiary();
+		$data['beneficiary'] = $b_obj->get_beneficiary_details($b_id);
+
+		$email = 'mothercare@sevasetu.org';
+		if($call_stats == 'Incorrect number')
+		{
+			$data['action'] = " has marked an Incorrect number status.";
```

Comment:
Move this to resources\lang\en\routes.php
All static strings are stored there.
Also, send in just a status from here, not the string. The controller's job is to send back logic. How that logic should be interepreted and shown to the user should be present on the view. Have this line in the view.